aurget

```diff
@@ -29,6 +30,9 @@ Usage: aurget [ -h | -S* [ --options ] [ -- ] <arguments> ]
         -Sp  <package>  print the PKGBUILD for <package>
         -Si  <package>  print extended info for <package>
 
+        --aur3          use the AUR3 ($AUR3)
```

Comment:
I think I would do this differently, maybe I can convince you:

- Have `--aur4` and `--noaur4`
- Don't rename `/AUR/AUR3/` or `/aur_foo/aur3_foo`

Put the conditionals at  the higher level only, (e.g.):

```sh
pkgbuild() {
  if aur4; then
    $AUR4/plain/PKGBUILD?...
  else
    $(aur_packages_url)/...
  fi
}
```

This should make for a smaller, clearer, and mostly-green diff. It should also prevent unnecessary "undo churn" (e.g. `/AUR3/AUR/`) when the aur4 behavior becomes default.

What do you think? I'd be happy to branch from your branch and try out this approach myself.